raster/r.horizon/main.c

```diff
@@ -57,94 +59,86 @@ module by Jaro Hofierka
 #define DISTANCE1(x1, x2, y1, y2) \
     (sqrt((x1 - x2) * (x1 - x2) + (y1 - y2) * (y1 - y2)))
 
-FILE *fp;
-
 const double pihalf = M_PI * 0.5;
 const double twopi = M_PI * 2.;
 const double invEarth = 1. / EARTHRADIUS;
 const double deg2rad = M_PI / 180.;
 const double rad2deg = 180. / M_PI;
-const double minAngle = DEG;
-
-const char *elevin;
-const char *horizon = NULL;
-const char *mapset = NULL;
-const char *per;
-char *shad_filename;
-char *outfile;
 
-struct Cell_head cellhd;
-struct Key_value *in_proj_info, *in_unit_info;
 struct pj_info iproj, oproj, tproj;
+float **z, **z100, **horizon_raster;
+int ll_correction = FALSE;
```

Comment:
You could declare `ll_correction` a bool and perhaps rename it to `[is|has]_ ll_correction`. 
You may use the standard  `false`  and `true` throughout the file instead of the macros.

Reply:
I kept the variable name, I see a value in keeping the original names to e.g. to compare the code with r.sun.